dist/leaflet.css

```diff
@@ -643,3 +643,13 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	margin-left: -12px;
 	border-right-color: #fff;
 	}
+
+/* Printing */
+	
+@media print {
+	/* Prevent printers from removing background-images of controls. */
+	.leaflet-control {
+		-webkit-print-color-adjust: exact;
+		              color-adjust: exact;

```

Comment:
There a spaces used instead of tabs

Reply:
Where is it documented what to use, and the tab size?

Reply:
@mourner? I only know it because it was / is often request on my PRs
https://github.com/Leaflet/Leaflet/pull/7816#pullrequestreview-817002397

Reply:
It's fine in this case because spaces are used for alignment, not indentation. Generally I think we'll convert the whole project to spaces at some point (and fix CRLF at the same time) — it brings more problems than it solves.